internal/operators/ocs/custom_manifest.go

```diff
@@ -0,0 +1,77 @@
+package ocs
+
+import (
+	"bytes"
+	"os"
+	"text/template"
+)
+
+// Subscription Manifest for testing internal builds
+func ocsCustomSubscription() (string, error) {
+	data := map[string]string{
+		"OPERATOR_NAMESPACE":         Operator.Namespace,
+		"OPERATOR_SUBSCRIPTION_NAME": Operator.SubscriptionName,
+	}
+
+	const ocsSubscription = `apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: "{{.OPERATOR_SUBSCRIPTION_NAME}}"
+  namespace: "{{.OPERATOR_NAMESPACE}}"
+spec:
+  installPlanApproval: Automatic
+  name: ocs-operator
+  source: ocs-catalogsource
+  sourceNamespace: openshift-marketplace`
+
+	tmpl, err := template.New("ocsSubscription").Parse(ocsSubscription)
+	if err != nil {
+		return "", err
+	}
+	buf := &bytes.Buffer{}
+	err = tmpl.Execute(buf, data)
+	if err != nil {
+		return "", err
+	}
+	return buf.String(), nil
+}
+
+// CatalogSource Manifest for testing internal builds
+func ocsCatalogSource() (string, error) {
+
+	image := "quay.io/rhceph-dev/ocs-registry:latest-stable-4.8"
```

Comment:
This will require the pull secret for this repo.

Reply:
yes, it will be required to add the Pull Secret if using internal builds